poetry/console/commands/install.py

```diff
@@ -63,13 +65,14 @@ def handle(self):
             # If this is a true error it will be picked up later by build anyway.
             return 0
 
-        self.line(
-            "  - Installing <info>{}</info> (<comment>{}</comment>)".format(
-                self.poetry.package.pretty_name, self.poetry.package.pretty_version
+        if not self.option("no-root"):
+            self.line(
+                "  - Installing <info>{}</info> (<comment>{}</comment>)".format(
+                    self.poetry.package.pretty_name, self.poetry.package.pretty_version
+                )
             )
-        )
 
-        if self.option("dry-run"):
+        if self.option("dry-run") or self.option("no-root"):
```

Comment:
The check on the `--no-root` option should be higher up (before the initialization of the `SDistBuilder`) 